ruleset.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
```

Comment:
This is not needed, you already have phpcs.xml

Reply:
removed